screens/ReportsScreen.tsx

```diff
@@ -7,7 +7,8 @@ import { Screen } from "../components/Screen";
 import Styles from '../constants/Styles';
 import { VictoryChart, VictoryLegend, VictoryPie } from 'victory-native';
 import { useQuery } from '@apollo/client';
-import { GetCategoriesDocument, GetCategoriesQuery } from '../components/generated';
+import { GetCategoriesDocument, GetCategoriesQuery, MonthBreakdownCategory } from '../components/generated';
+import ByCategory from '../components/GraphDisplays/ByCategory';
```

Comment:
change this to "import ByCategory from '../components/GraphDisplays/byCategory';"

lower case b